### PR TITLE
Fix translatable strings 2

### PR DESCRIPTION
**Pull request by Scrumplex**

### CODE REVIEW

**Comment by d-513** [comment]
It feels werid. Perhaps we could make it a build-type option?
Waiting for GHA build so dont really know if that was the changed thing, but in the about page we should probably mention it by name

**Comment by Scrumplex** [comment]
> It feels werid. Perhaps we could make it a build-type option? Waiting for GHA build so dont really know if that was the changed thing, but in the about page we should probably mention it by name

Oh wait. I assumed that would be the name of the first tab, but that might actually be the name of the launcher. Imma fix that

**Comment by DioEgizio** [comment]
But why

**Comment by Scrumplex** [comment]
> But why

To make it easier for people to make their own launcher based off PolyMC

File: launcher/ui/dialogs/AboutDialog.ui [hunk 1]
```diff
@@ -80,7 +80,7 @@
       </font>
      </property>
      <property name="text">
-      <string notr="true">PolyMC</string>
+      <string notr="true">Launcher</string>
```
Comment (Scrumplex) [inline launcher/ui/dialogs/AboutDialog.ui:83]: This will be replaced programmatically anyway, so this could also say `pee-pee poo-poo`

**Comment by DioEgizio** [comment]
> > But why
> 
> To make it easier for people to make their own launcher based off PolyMC

Oh yeah ok

File: launcher/ui/dialogs/AboutDialog.ui (hunk 1 above)
Comment (d-513) [reply launcher/ui/dialogs/AboutDialog.ui:83]: Ok

**Review by d-513 — APPROVED** [review]

**Review by txtsd — APPROVED** [review]

**Review by ZekeZDev — APPROVED** [review]

**Pull request merged**